pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -1652,6 +1648,7 @@ private static void fixColumnName(String rawTableName, Expression expression, Ma
    * Returns the actual column name for the given column name for:
    * - Case-insensitive cluster
    * - Column name in the format of [table_name].[column_name]
+   * - Column name in the format of [database_name].[table_name].[column_name]
```

Comment:
I don't think we need to support such complicated column name (at least for database isolation purpose)

Reply:
Ack. Will skip this handling.